tests/components/enphase_envoy/test_sensor.py

```diff
@@ -41,7 +44,7 @@ async def test_sensor(
     entity_registry: er.EntityRegistry,
     config_entry: MockConfigEntry,
     snapshot: SnapshotAssertion,
-    setup_enphase_envoy_sensor,
+    setup_enphase_envoy_sensor: None,
```

Comment:
maybe pytest.mark.usefixture...

Reply:
Doesn't work here... it seems the order is important, and it needs to come after the other fixtures.